configs/test/bigquery/datasets.yaml

```diff
@@ -225,3 +225,29 @@ resources:
             type: FLOAT
           - name: engine
             type: STRING
+  - name: cross-pollination-statistics-table
+    type: gcp-types/bigquery-v2:tables
+    properties:
+      datasetId: $(ref.main-dataset.datasetReference.datasetId)
+      tableReference:
+        tableId: cross-pollination-statistics
+      timePartitioning:
+        type: DAY
+      schema:
+        fields:
+        - name: project_qualified_name
+          type: STRING
+        - name: tagged
```

Comment:
would it make sense to record the tags as well? I guess they can change over time, so having explicit tags recorded might be useful knowledge

Reply:
Same as with sources, there are multiple tags for each target. Should every tag be listed?